cmd/kubectl-testkube/commands/webhooks/create.go

```diff
@@ -52,6 +54,7 @@ func NewCreateWebhookCmd() *cobra.Command {
 	cmd.Flags().StringVarP(&name, "name", "n", "", "unique webhook name - mandatory")
 	cmd.Flags().StringArrayVarP(&events, "events", "e", []string{}, "event types handled by executor e.g. start-test|end-test")
 	cmd.Flags().StringVarP(&uri, "uri", "u", "", "URI which should be called when given event occurs")
+	cmd.Flags().StringToStringVarP(&labels, "label", "l", nil, "label key value pair: --label key1=value1")
```

Comment:
should we be consistent when naming this (selectors and labels)?

Reply:
yeah internally those are selectors (AFAIK) but from user point of view we want to sell them as labels :)

Reply:
So, generally selector is an expression on labels, and for any Get, Delete, Run we use internally (in api) native k8s selectors(in api you can pass "key1 in (value1, value2)") , for Create and Update we simply use labels, but for cli users as @exu mentioned for simplicity, it's always labels